src/mlpack/methods/ann/quantization/quantization_utils.hpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file quantization_utils.hpp
+ * @author Mark Fischinger
+ *
+ * Utility functions and classes for quantization in neural networks.
+ */
+#ifndef MLPACK_METHODS_ANN_QUANTIZATION_QUANTIZATION_UTILS_HPP
+#define MLPACK_METHODS_ANN_QUANTIZATION_QUANTIZATION_UTILS_HPP
+
+#include <mlpack/prereqs.hpp>
```

Comment:
If you can, use `base.hpp` as the include.  The reason for this is that including as little as possible helps compilation times.